staticcheck.conf

```diff
@@ -0,0 +1 @@
+checks = ["all", "-ST1000", "-ST1003", "-ST1016", "-ST1023"]
```

Comment:
I think this is not necessary maybe c/p from the jwt repo

Reply:
Removed